pulp_python/app/models.py

```diff
@@ -1,35 +1,73 @@
-from gettext import gettext as _
 from logging import getLogger
 
 from django.db import models
 
-from pulpcore.plugin.models import (Artifact, Content, ContentArtifact, RemoteArtifact, Importer,
-                                    ProgressBar, Publisher, RepositoryContent, PublishedArtifact,
-                                    PublishedMetadata)
-from pulpcore.plugin.tasking import Task
+from pulpcore.plugin.models import Content, Importer, Publisher
 
 
 log = getLogger(__name__)
 
 
-class PythonContent(Content):
+class Classifier(models.Model):
     """
-    The "python" content type.
+    Custom tags for classifier
 
-    Define fields you need for your new content type and
-    specify uniqueness constraint to identify unit of this type.
+    Fields:
 
-    For example::
+        name (models.TextField): The name of the classifier
 
-        field1 = models.TextField()
-        field2 = models.IntegerField()
-        field3 = models.CharField()
+    Relations:
 
-        class Meta:
-            unique_together = (field1, field2)
+        pythonpackagecontent (models.ForeignKey): The task this tag is associated with
+
+    """
+
+    name = models.TextField()
+    pythonpackagecontent = models.ForeignKey("PythonPackageContent", related_name="classifiers",
+                                             related_query_name="classifier")
+
+
+class PythonPackageContent(Content):
+    """
+    A Content Type representing Python's Distribution Package as
+    defined in pep-0426
     """
 
     TYPE = 'python'
+    filename = models.TextField(unique=True, db_index=True, blank=False)
+    path = models.TextField(blank=False)
+    packagetype = models.TextField(blank=False)
+    name = models.TextField(blank=False)
+    version = models.TextField(blank=False)
+
+    # Note that this field is not contained in PyPI nor Warehouse
```

Comment:
Could you clarify this comment? This field isn't returned by the json API, but it is a part of the PKG-INFO/METADATA .

Reply:
neither warehouse nor PyPI stores this information (that I could tell)